h2o-py/tests/testdir_misc/pyunit_explain.py

```diff
@@ -11,6 +11,28 @@
 from h2o.estimators import H2OGradientBoostingEstimator
 from h2o.explanation._explain import H2OExplanation
 
+
```

Comment:
Would call this pyunit_pubdev_8013  and would put inside of thestdir_jira instead of testdir_misc to be inline with the second test.